app/assets/stylesheets/styles.scss

```diff
@@ -40,25 +46,4 @@ li {
 .side-panel-image {
   text-align: center;
   margin: 1.5em;
-}
-
-.favorite-button, .unfavorite-button {
```

Comment:
These styles are used on the School#show page for the heart icon.  Would you add them back in or modify them (as you see fit)?